src/test/java/duke/StorageTest.java

```diff
@@ -0,0 +1,15 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
I like how your ordering of import statements stay consistent throughout the work.